### DISCUSSION TITLE
displayName

### DISCUSSION THREAD

**Discussion by HerbertUnterberger** [msegzda/homebridge-nordpool-baltics#19] [Show and tell]
Hi,
I try to define the displayName in HomeKit as you defined it for Homebridge.
not just in Settings.

I am a newbee in TypeScript and object calling is a bit different to python.

Thx for your great work.

Herbert, your ausrtian fork

**Comment by msegzda** [comment]
Im sorry, i do not quite understand the question. Can you please describe
what are you trying to do within your fork?

**Reply by HerbertUnterberger** [reply]
For using the accessories in shortcuts or Home app, I try to give the same naming as you defined for homebridge.

in Funktions.ts

    // init light sensor for current price
    this.service.currently = this.accessory.getService('Nordpool_currentPrice') || this.accessory.addService(
      this.platform.Service.LightSensor, 'Nordpool_currentPrice', 'currentPrice');

Nordpool_currentPrice is clear named, but the  in hpa the comes from 

setting.ts

// main device(s)
export const devices = [
  {
    UniqueId: 'JKGhJH654*87pDE',
    displayName: 'Nordpool',
  },
];


platforms.ts

  // create a new accessory
        const accessory = new this.api.platformAccessory(device.displayName, uuid);


so all devices are called: Nordpool

I love to see in HomeApp : Nordpool_currentPrice

**Comment by msegzda** [comment]
This is known Apple homekit api issue. You will be able to see the defined
names (as default) when you edit accessory name on Home app. Then you know
which is which

**Reply by HerbertUnterberger** [reply]
thx
thats why I couldn't find it.
